src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
 
-### Step 2: Name your NRQL condition [#step2-nrql]
+  Next, you will need to define what defines a **critical anomaly threshold violation**. For this first alert, we recommend using our default settings and adjusting to your needs as necessary. So, leave the settings to open an incident "when a query returns a value deviates from the predicted value: **for at least five minutes** by **3 standard deviations**".
 
-Give your NRQL condition a name that's meaningful to you.
+  Learn more about anomalies in our anomaly (documentation)[/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/].
 
-### Step 3: Set your condition thresholds [#step3-nrql]
+  <img
+    title="Configure your anomaly thresholds"
+    alt="A screenshot demonstrating how to configure custom anomalies"
+    src={alertsConfigureYourAnomalyThresholds}
+  />
 
-Set critical and warning thresholds to determine when you're notified about your environment's performance. You must configure at least one threshold.
+</Collapser>
 
-* A critical threshold is optional for your alerts condition.  
+  <Collapser
+    id="static-threshold"
+    title="Static threshold"
+  >
+    Unlike anomaly thresholds, a static threshold doesn't look at your data set as a whole and determine what behavior is unusual based on your systems history, instead a static threshold will open an incident whenever your system behaves differently than the conditions that **you set**. Static alert thresholds are much more customizable and we recommend them if you've a strong sense of your data and what you're looking for. 
 
-* A [warning threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition/#threshold-levels) is optional.
+    Learn more about our static alert conditions in our [NRQL docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#threshold-types).
+  </Collapser>
+  
+  <Collapser
+    id="lost-signal"
+    title="Lost signal threshold (optional)"
+  >
+  Sometimes an incoming signal can be lost and it's important to understand if this is simply a delay or if it's an indication of a wider problem. Our **loss signal threshold** lets New Relic knows how many seconds the system should wait, from the time the last data point was detected, before considering the signal to be lost. If the signal does not return before the threshold that you set, you can choose one of two things: receive a notification about the lost signal, or you can close any related open violations if the lost signal was expected.
```

Comment:
"Knows" should be "know" I think. I don't think you need a comma after "should wait" or "detected"

Reply:
done